vulture/core.py

```diff
@@ -362,6 +362,9 @@ def visit_arg(self, node):
         """Function argument. Python 3 only. Has lineno since Python 3.4"""
         self._define_variable(node.arg, node, confidence=100)
 
+    def visit_AsyncFunctionDef(self, node):
+        self.visit_FunctionDef(node)
```

Comment:
`return self.visit_FunctionDef(node)` (in case the called function ever returns anything in the future).